arch/arm/arm64/include/uk/asm/ctx.h

```diff
@@ -56,6 +56,14 @@
 		__sp__;					\
 	})
 
+#define ukarch_rstack_push_packed(sp, value)	\
```

Comment:
Not sure whether GitHub it's just not showing tabs correctly, but the indentation looks off for the backslash.